### PR TITLE
Initial outline for Diagnostics and Therapeutics sections

### PR DESCRIPTION
**Pull request by rando2**
Lots of questions to be filled in later!

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+
+### Treatment of Symptoms
+COVID-19 is characterized by...
+The most severe and concerning symptoms are typically...
+The symptoms most often regarded as the proximal cause of death from COVID-19 are...
+Other diseases with similar symptoms include XYZ but these diseases may be different because...
+
+So far, some strategies for reducing the severity of symptoms have included...
+
+#### Relevant Mechanisms and Possible Pharmaceutical Interventions
```
Comment (cgreene) [inline content/20.treatments.md]: Maybe make clear that these are mechanisms for symptom management to guide contributors? I'm imagining that after this you have a section for each potential therapeutic that you want to cover in this class of interventions.

File: content/20.treatments.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+Antivirals have been used to treat XYZ diseases through [what mechanisms or approaches are used?]
+
+#### Antivirals and Coronoviruses
+[Have antivirals been used to treat other coronaviruses? What went well/badly? How do these interventions work?]
+
+#### Antiviral Targeting of SARS-CoV-2
+[What efforts are underway to develop antivirals for SARS-CoV-2? Is anything expected to be particularly challenging about this virus?]
+[What are possible ways to target SARS-CoV-2 that are being proposed, developed, or tested?]
+
+#### Data-Driven Analysis and Identification of Antivirals
```
Comment (cgreene) [inline content/20.treatments.md]: Similar to above. I would skip this for reasons described.

File: content/20.treatments.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+### Antivirals
+Antivirals are an emerging category of drugs.
+Unlike antibiotics, they do not kill viruses.
+Rather, they inhibit the proliferation of a virus.
+Antivirals have been used to treat XYZ diseases through [what mechanisms or approaches are used?]
+
+#### Antivirals and Coronoviruses
+[Have antivirals been used to treat other coronaviruses? What went well/badly? How do these interventions work?]
+
+#### Antiviral Targeting of SARS-CoV-2
```
Comment (cgreene) [inline content/20.treatments.md]: I'd probably make these types of synthesis questions all part of the section without any separation and then have the subsections be the names of potential therapies plus the evidence around them.

File: content/20.treatments.md [hunk 4]
```diff
@@ -1,2 +1,49 @@
...
+### Vaccines
+Vaccines, widely recognized as one of the most significant advances in human health during the 20th century, can be used to bolser both individual and herd immunity to a virus by promoting the development of antibodies without infection.
+[Are vaccines available for other coronaviruses or related viral illnesses?]
+
+#### Strategies and Challenges to Vaccine Development
+[What are some of the challenges to developing a vaccine? What needs to be taken into account about how the virus works?]
+[Are there any challenges or opportunities unique to coronaviruses and/or SARS-CoV-2?]
+[What are some approaches being tested or considered?]
+
+#### Big Data and Vaccines
```
Comment (cgreene) [inline content/20.treatments.md]: Similarly to above :)

File: content/20.treatments.md [hunk 5]
```diff
@@ -1,2 +1,49 @@
...
+The most severe and concerning symptoms are typically...
+The symptoms most often regarded as the proximal cause of death from COVID-19 are...
+Other diseases with similar symptoms include XYZ but these diseases may be different because...
+
+So far, some strategies for reducing the severity of symptoms have included...
+
+#### Relevant Mechanisms and Possible Pharmaceutical Interventions
+[Given what we know about the mechanisms of the virus and why it produces the symptoms we see, are there drugs or categories of intervention that might be relevant?]
+
+#### Data-Driven Prediction of Interventions
```
[the remark this answers is not in the document]
Comment (agitter) [reply content/20.treatments.md]: That scope could grow large quickly, so I think it is too much for me to take on. [ChemRxiv](https://chemrxiv.org/search?q=sars-cov-2&searchMode=1) has 18 relevant preprints and counting plus many more manuscripts elsewhere.

A heuristic could be to focus on repurposing or on those data-driven predictions that have advanced to some form of lab testing.

File: content/10.diagnostics.md [hunk 6]
```diff
@@ -1,2 +1,17 @@
-##Diagnostics  {.page_break_before} 
-This section will include a review of the available information about diagnostics.
+## Diagnostics  {.page_break_before} 
+### Current Strategies for Diagnosing COVID-19 and Similar Viral Infections
+Given the heterogeneity of symptom presentation across patients with COVID-19, the development of standardized protocols for testing samples for SARS-CoV-2 is urgent.
+Following the release of the genetic sequence of the virus by Chinese officials on January 12, 2020, the first tests for detecting the virus were released on XX, 2020.
+These tests used the following approach to identify the active virus in patient samples...
+However, many countries have struggled to acquire the tests required to keep pace with the epidemic.
+[Why is it so difficult to scale up testing? What are some of the considerations?]
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/10.diagnostics.md]: @cgreene I think opening an issue would be great, it seems like there should be some discussion around how best to approach this. For now, I made a first pass at incorporating your suggestions in the existing outline in 7cde06436d55cfac5543e187eda4829d2d65bcbc . I was thinking of making a new category issue, such as "request for expertise" (or maybe "discussion"?) and adding epidemiology as a label. A) Does that seem like the right direction to go, and B) Are there other fields I should tag besides epidemiology?

[the remark this answers is not in the document]
Comment (cgreene) [reply content/10.diagnostics.md]: Maybe "Discussion of Additional Sections" or something as the title, and then we could refer folks there and see if we get a critical mass?

**Comment by rando2** [comment]
@rmrussell we would love your feedback on how to improve the outlines! I'm working on implementing these changes above and will hopefully have them online soon. In the meantime, if you want to start thinking about what changes you would make to the outline, I can give you instructions on how to modify my version to add suggestions!

File: content/20.treatments.md [hunk 7]
```diff
@@ -1,2 +1,49 @@
 ## Therapeutics {.page_break_before}
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/20.treatments.md]: Nope, I think I changed my mind partway through! Hopefully addressed in 5fcf8e9066ff792420aac73d18adbbee27cd496f and in the forthcoming PR for 20.Therapeutics (I'll edit to add link when it's up)

File: content/20.treatments.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/20.treatments.md]: More than happy to drop these sections! I misunderstood the scope of the objective initially.

File: content/10.diagnostics.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/10.diagnostics.md]: #22

File: content/20.treatments.md [hunk 8]
```diff
@@ -1,2 +1,49 @@
...
+So far, some strategies for reducing the severity of symptoms have included...
+
+#### Relevant Mechanisms and Possible Pharmaceutical Interventions
+[Given what we know about the mechanisms of the virus and why it produces the symptoms we see, are there drugs or categories of intervention that might be relevant?]
+
+#### Data-Driven Prediction of Interventions
+[The machine learning community has been called upon to try to predict possible interventions. What are they doing and what are they finding about treating symptoms?]
+
+### Antivirals
+Antivirals are an emerging category of drugs.
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/20.treatments.md]: Sounds good to me!

File: content/20.treatments.md (hunk 3 above)
Comment (rando2) [reply content/20.treatments.md]: May have gone overboard but it's now one section per topic (symptoms, antivirals,  vaccines) and a direction to make new categories as appropriate

**Comment by rando2** [comment]
I'm merging this so we can move forward with changes from subject matter experts.

File: content/10.diagnostics.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/10.diagnostics.md]: My earlier comment is showing up after your comment, so it looks like I completely ignored your suggestion.... this suggestion has now been implemented as #36

**Comment by rando2** [comment]
Something went wrong and not all of these changes were in the repository. I manually checked 10.Diagnostics and it looks like the correct version. I'm not sure what went wrong with 20.Therapeutics but some of the changes are waiting for public health feedback in #20 , and the straightforward changes are now implemented in #42

**Pull request merged**